scripts/endToEndExtraction/create_traces.sh

```diff
@@ -1,22 +1,25 @@
-BASE_PATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 || exit ; pwd -P )"
+#!/usr/bin/env bash
+set -eu
+
+BASE_PATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 && pwd -P )"
 
 mkdir -p build
-cd build || exit
```

Comment:
Why were these removed?

Reply:
Because I added `set -eu` at the top of the script so it will fail automatically if `cd` fails.